roles/navidrome/defaults/main.yml

```diff
@@ -0,0 +1,144 @@
+# Author(s):        finch, kowalski                                      #
+# URL:              https://github.com/saltyorg/Community                #
+# --                                                                     #
+##########################################################################
+#                   GNU General Public License v3.0                      #
+##########################################################################
+---
+################################
+# Basics
+################################
+
+navidrome_name: navidrome
+
+################################
+# Paths
+################################
+
+navidrome_paths_folder: "{{ navidrome_name }}"
+navidrome_paths_location: "{{ server_appdata_path }}/{{ navidrome_paths_folder }}"
+navidrome_paths_folders_list:
+  - "{{ navidrome_paths_location }}"
+
+################################
+# Web
+################################
+
+navidrome_web_subdomain: "{{ navidrome_name }}"
+navidrome_web_domain: "{{ user.domain }}"
+navidrome_web_port: "4533"
+navidrome_web_url: "{{ 'https://' + navidrome_web_subdomain + '.' + navidrome_web_domain
+                    if (reverse_proxy_is_enabled)
+                    else 'http://localhost:' + navidrome_web_port }}"
+
+################################
+# DNS
+################################
+
+navidrome_dns_record: "{{ navidrome_web_subdomain }}"
+navidrome_dns_zone: "{{ navidrome_web_domain }}"
+
+################################
+# Traefik
+################################
+
+navidrome_traefik_middleware: "{{ traefik_default_middleware }}"
+navidrome_traefik_certresolver: "{{ traefik_default_certresolver }}"
+navidrome_traefik_enabled: true
+
+################################
+# Docker
+################################
+
+# Container
+navidrome_docker_container: "{{ navidrome_name }}"
+
+# Image
+navidrome_docker_image_pull: true
+navidrome_docker_image_tag: "latest"
+navidrome_docker_image: "deluan/navidrome:{{ navidrome_docker_image_tag }}"
+
+# Ports
+navidrome_docker_ports_defaults:
+  - "{{ navidrome_web_port }}"
+navidrome_docker_ports_custom: []
+navidrome_docker_ports: "{{ navidrome_docker_ports_defaults
+                            + navidrome_docker_ports_custom
+                         if (not reverse_proxy_is_enabled)
+                         else [] + navidrome_docker_ports_custom }}"
+
+# Envs
+navidrome_docker_envs_default: {}
+navidrome_docker_envs_custom: {}
+navidrome_docker_envs: "{{ navidrome_docker_envs_default
+                           | combine(navidrome_docker_envs_custom) }}"
+
+# Commands
+navidrome_docker_commands_default: []
+navidrome_docker_commands_custom: []
+navidrome_docker_commands: "{{ navidrome_docker_commands_default
+                               + navidrome_docker_commands_custom }}"
+
+# Volumes
+navidrome_docker_volumes_default:
+  - "{{ navidrome_paths_location }}:/data"
+  - "{{ navidrome.musicfolder }}:/music"
```

Comment:
I think we need to avoid doing mappings like that, if people really want to add it they can do so through the inventory system.